.github/mergify.yml

```diff
@@ -0,0 +1,68 @@
+pull_request_rules:
+  - name: Automatic merge when all checks pass and the PR is approved
+    conditions:
+      - "#approved-reviews-by>=1"
+      - "-draft"
+      - "status-success=basic-checks"
+      - "status-success=end-to-end (smoke)"
+      - "status-success=end-to-end (es)"
+      - "status-success=end-to-end (cassandra)"
+      - "status-success=end-to-end (streaming)"
+      - "status-success=end-to-end (examples1)"
+      - "status-success=end-to-end (examples2)"
+      - "status-success=end-to-end (generate)"
+      - "status-success=end-to-end (es-otel)"
+      - "status-success=end-to-end (streaming-otel)"
+      - "status-success=DCO"
+      - "status-success=WIP"
+      - "status-success=codecov/patch" # we don't require those, but we might want to manually check before automatically merging
+      - "status-success=codecov/project"
+    actions:
+      merge:
+        method: squash
+        commit_message: title+body
+        bot_account: jaegerci-bot
+  - name: Kafka changes
+    conditions:
+    - files~=kafka
+    actions:
+      assign:
+        users:
+          - rubenvp8510
```

Comment:
Are you ok in owning everything related to Kafka?

Reply:
+1